zeth-contracts/contracts/Pairing.sol

```diff
@@ -51,7 +59,7 @@ library Pairing {
         assembly {
             // bn256Add precompiled: https://github.com/ethereum/go-ethereum/blob/master/core/vm/contracts.go#L57
             // Gas cost: 500 (see: https://github.com/ethereum/go-ethereum/blob/master/params/protocol_params.go#L84)
```

Comment:
The lines have changed because of the instanbul update:
https://github.com/ethereum/go-ethereum/blob/3bb6815fc1ade61e85b7039cf2e08a0871bb47e4/params/protocol_params.go#L124

we should also update to istanbul costs